### ISSUE TITLE
Vampires Dawn II: No visual indication when using a healing spell

### ISSUE THREAD

**Issue by Ghabry** [EasyRPG/Player#2479]
In VD2 the actors do not flash when selecting a target for Skill "Schwache Heilung" making it almost impossible to target the correct one.

[Save05.zip](https://github.com/EasyRPG/Player/files/6141175/Save05.zip)
Fight the slime to the left of you. NOTE: The scrolling map is because this is a very old EasyRPG Player save (is fixed by changing the map once).

@rueter37

**Comment by rueter37** [comment]
@Ghabry check out this branch: https://github.com/rueter37/player/tree/battleanimation-sprites. I'm going to PR this one after #2399 has been merged. This is the successor of #2328 and plans on fixing issues in relation to battle animation sprites. This covers this issue as well.

**Comment by Ghabry** [comment]
btw if you want you can already open your branch as a PR (set it to Draft when opening it). Then we can already start reviewing it.

That it is based on fmatthews is no problem. When that is merged you simply rebase and then yours gets the turn.

**Comment by rueter37** [comment]
I have added a draft PR now (#2482).

**Comment by rueter37** [comment]
@Ghabry Can you test if [this branch](https://github.com/rueter37/Player/tree/battleanimation-sprites-flashes) which tries to fix the flash effects on battle animation sprites works for you? If yes, I would add a new PR after #2482 is merged in or we can put it on top of #2482.

**Comment by rueter37** [comment]
I have added a draft PR now (#2505) which should hopefully fix this issue.
